src/Integration.Vsix.UnitTests/ReferencesTests.cs

```diff
@@ -30,10 +29,11 @@ namespace SonarLint.VisualStudio.Integration.UnitTests
     public class ReferencesTests
     {
         [TestMethod]
-        public void MicrosoftVisualStudioCodeAnalysis_EnsureCorrectVersion()
+        public void MicrosoftVisualStudioVCProjectEngine_EnsureCorrectVersion()
         {
             var codeAnalysisAssemblyVersion = AssemblyHelper.GetVersionOfReferencedAssembly(
-                    typeof(SonarLint.VisualStudio.Integration.Binding.BindingController), "Microsoft.VisualStudio.CodeAnalysis");
```

Comment:
It isn't obvious from the GitHub diff, but I moved this test from `Integration.UnitTests` to `Integration.VSIX.UnitTests` and changed it to test a different assembly.

The VS ruleset assemblies are not included in the VS SDK. Also, we needed different versions for VS2019 and VS2022. So, we had checked-in binaries under the `third-parties` folder (deleted below) that we referenced in the projects.

This test checked that SLVS 2019 referenced the VS2019 RuleSet assembly and that SLVS 2022 referenced the VS2022 RuleSet assembly.

We no longer need this test to check the ruleset assembly. However, there is another third-party assembly similar to the RuleSet assembly that we didn't have a test for, namely `Microsoft.VisualStudio.VCProjectEngine`. So this test now validates that the correct version of that assembly is being used instead.